src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
```

Comment:
```suggestion
```
remove empty lines

Reply:
If Yoour's comment about this work is really true, I'm happy to read these words. But I don't think so, because during making this program I had too many stupid mistakes and asked too many stupid questions. But I'll do as much as possible and even unpossible to become better in future.